cmake/Gems.cmake

```diff
@@ -261,7 +263,7 @@ function(ly_enable_gems_delayed)
     get_property(targets_with_variants GLOBAL PROPERTY LY_TARGETS_WITH_GEM_VARIANTS)
     # Query the projects that have made calls to ly_enable_gems
     get_property(enable_gem_projects GLOBAL PROPERTY LY_DELAYED_ENABLE_GEMS)
-
+    
```

Comment:
Superfluous whitespace has been added here.

Reply:
I thought this matched the style of the rest of file better but I can remove it if you prefer.

Reply:
Oh you mean the tab, yeah I'll clean that up.